fleetshard/pkg/runtime/runtime.go

```diff
@@ -245,18 +246,27 @@ func (r *Runtime) deleteStaleReconcilers(list *private.ManagedCentralList) {
 }
 
 func (r *Runtime) upgradeOperator(list private.ManagedCentralList) error {
+	var desiredOperators []operator.DeploymentConfig
 	var desiredOperatorImages []string
 	for _, central := range list.Items {
+		// TODO: read GitRef ManagedCentral
```

Comment:
nit: sounds to me like the `GitRef` is part of a ManagedCentral.

```suggestion
		// TODO: read GitRef ManagedCentral list call
```